apps/frontend/src/components/TimeCardPage/CellTypes/CommentModals/ShowCommentModal.tsx

```diff
@@ -51,10 +53,22 @@ const saveEditedComment = (
   // previous comment edited over so set it to deleted
   prevComment.State = CellStatus.Deleted
   setComments(getAllActiveCommentsOfType(typeOfComment, [...comments, newComment]));
-  // TODO: save to DB
+  
+  var rowToMutate = row.Comment; 
+  if (rowToMutate === undefined) {
+      rowToMutate = comments;
+  }
+  // Value is null, so mark it as undefined in our processing 
+  rowToMutate = setComments([...comments, newComment]);
+  console.log(rowToMutate)
+  
+  //Triggering parent class to update its references here as well 
+  updateField("Comment", rowToMutate); 
 };
 
 const deleteComment = (
+  updateField: Function,
+  row: RowSchema,
```

Comment:
I think it's probably overkill to be passing in the *entire* shift row here. Just pass in the parts you need - in this case, the current comments (`row.Comment`). Same with the `saveEditedComment` function above.

Reply:
e1f9579a42a756a9d442c7797c9d3b9ef29e7537